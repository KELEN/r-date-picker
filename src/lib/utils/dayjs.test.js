import {
  getDateArray,
} from './dayjs';

describe('css prefix', () => {
  test('get current month array', () => {
    const dateArray = getDateArray();
    expect(dateArray).toMatchSnapshot();
  });

  test('get custom month array', () => {
    const dateArray = getDateArray('2021-04');
    expect(dateArray).toMatchSnapshot();
  });
});
