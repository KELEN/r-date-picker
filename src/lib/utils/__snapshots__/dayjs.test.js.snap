// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`css prefix get current month array 1`] = `
Array [
  Array [
    Object {
      "date": "2021-04-25T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-26T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-27T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-28T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-29T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-30T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-01T16:00:00.000Z",
    },
  ],
  Array [
    Object {
      "date": "2021-05-02T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-03T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-04T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-05T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-06T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-07T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-08T16:00:00.000Z",
    },
  ],
  Array [
    Object {
      "date": "2021-05-09T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-10T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-11T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-12T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-13T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-14T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-15T16:00:00.000Z",
    },
  ],
  Array [
    Object {
      "date": "2021-05-16T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-17T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-18T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-19T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-20T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-21T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-22T16:00:00.000Z",
    },
  ],
  Array [
    Object {
      "date": "2021-05-23T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-24T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-25T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-26T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-27T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-28T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-29T16:00:00.000Z",
    },
  ],
  Array [
    Object {
      "date": "2021-05-30T16:00:00.000Z",
    },
    Object {
      "date": "2021-06-01T15:59:59.999Z",
    },
    Object {
      "date": "2021-06-02T15:59:59.999Z",
    },
    Object {
      "date": "2021-06-03T15:59:59.999Z",
    },
    Object {
      "date": "2021-06-04T15:59:59.999Z",
    },
    Object {
      "date": "2021-06-05T15:59:59.999Z",
    },
  ],
]
`;

exports[`css prefix get custom month array 1`] = `
Array [
  Array [
    Object {
      "date": "2021-03-28T16:00:00.000Z",
    },
    Object {
      "date": "2021-03-29T16:00:00.000Z",
    },
    Object {
      "date": "2021-03-30T16:00:00.000Z",
    },
    Object {
      "date": "2021-03-31T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-01T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-02T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-03T16:00:00.000Z",
    },
  ],
  Array [
    Object {
      "date": "2021-04-04T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-05T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-06T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-07T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-08T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-09T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-10T16:00:00.000Z",
    },
  ],
  Array [
    Object {
      "date": "2021-04-11T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-12T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-13T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-14T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-15T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-16T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-17T16:00:00.000Z",
    },
  ],
  Array [
    Object {
      "date": "2021-04-18T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-19T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-20T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-21T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-22T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-23T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-24T16:00:00.000Z",
    },
  ],
  Array [
    Object {
      "date": "2021-04-25T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-26T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-27T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-28T16:00:00.000Z",
    },
    Object {
      "date": "2021-04-29T16:00:00.000Z",
    },
    Object {
      "date": "2021-05-01T15:59:59.999Z",
    },
    Object {
      "date": "2021-05-02T15:59:59.999Z",
    },
  ],
  Array [
    Object {
      "date": "2021-05-03T15:59:59.999Z",
    },
    Object {
      "date": "2021-05-04T15:59:59.999Z",
    },
    Object {
      "date": "2021-05-05T15:59:59.999Z",
    },
    Object {
      "date": "2021-05-06T15:59:59.999Z",
    },
    Object {
      "date": "2021-05-07T15:59:59.999Z",
    },
    Object {
      "date": "2021-05-08T15:59:59.999Z",
    },
  ],
]
`;
